src/index.html

```diff
@@ -146,6 +146,251 @@
          document.getElementById("main").style.marginLeft= "0";
      }
     </script>
+
+    <svg>
+      <defs id="symbols">
```

Comment:
We now have duplicate symbols, in the `img/*_symbol.svg` and here. Best to have them in a single place and reference.

Reply:
We are using both because the "img/*_symbol.svg" seem easier to use as buttons but on the other hand seems not possible to color them in the d3 code. I need extra time to play a little bit more with svg/CSS